### ISSUE TITLE
Allow sorting search by document date, not just filed case date

### ISSUE THREAD

**Issue by ikeboy** [freelawproject/recap#285]
Currently, courtlistener.com search allows sorting by date the case was filed, along with filters for the filed date. However, there's no way to search for _documents_ filed recently but perhaps in older cases. I want to search for all recent documents containing various keywords, but can't. Seems like a relatively simple addition to the website.

**Comment by mlissner** [comment]
Well, the field name we need to look at is `entry_date_filed`', which is described in our search engine as:

```
    <field name="entry_date_filed"
           type="date"
           indexed="true"
           stored="true"
           multiValued="false"/>
```

Unfortunately, that doesn't have `docValues=true`, which supposedly can make sorting on the field not so great, but it's unclear without testing exactly how bad that'd be.

As far as the front end goes, I thought I might just be able to use the field and have it magically work, so I tried this:

https://www.courtlistener.com/?type=r&q=cucumber&type=r&order_by=entry_date_filed%20desc

Alas, it throws an error, so no luck for us there. Gotta dig in a bit to figure out what the error is.

I remember debating whether this was a useful feature back when we made the search engine originally, and for some reason I decided it wasn't that important. It does seem like it should be really easy though, so I don't know what my past-self was thinking. Perhaps I have a note we'll find when/if I dig into this more deeply. 

Anyway, bottom line: Backend seems like it might be OK; front end doesn't work for unknown reason. Worth investigating further.

**Comment by ikeboy** [comment]
Looking at search_utils.py, it converts the existing date search to use the entry_date_filed field anyway

       if sort_string == "dateFiled asc":
           return "entry_date_filed asc"
       elif sort_string == "dateFiled desc":
           return "entry_date_filed desc"
